kDrive/UI/Controller/Files/Upload/UploadQueueViewController.swift

```diff
@@ -58,6 +59,7 @@ final class UploadQueueViewController: UIViewController {
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
         MatomoUtils.track(view: [MatomoUtils.Views.uploadQueue.displayName, "Main"])
+        tableView.reloadData()
```

Comment:
If we need to `reloadData` on `viewDidAppear` we might have an observation issue somewhere.
We should try to call it only when we have to, I'll check it.